examples/matmul_py/task.yml

```diff
@@ -6,6 +6,20 @@ files:
   - {"name": "utils.py", "source": "../utils.py"}
   - {"name": "reference.py", "source": "reference.py"}
   - {"name": "eval.py", "source": "../eval.py"}
+  - {"name": "pytorch_ref.py", "source": "pytorch_ref.py"}
```

Comment:
Milestones shouldn't be added here. These files get sent to the runner for _every_ submission.

Reply:
ohh is there another place to config problems or not really?